examples/http-gateway/cloudflare/src/index.ts

```diff
@@ -0,0 +1,40 @@
+import { HttpGatewayClient } from '@dfinity/http-gateway';
+import { HttpAgent } from '@dfinity/agent';
+import RESPONSE_VERIFICATION_WASM from '@dfinity/response-verification/dist/web/web_bg.wasm';
```

Comment:
I had issues with loading the WASM when the HTTP Gateway library did it internally. This should work fine in other environments, but Cloudflare doesn't like it for some reason. Loading it explicitly inside the worker and passing it as a parameter works well, although I really don't like it.